api/db/database-builder/factory/build-certification-center-membership.js

```diff
@@ -9,14 +9,17 @@ const buildCertificationCenterMembership = function ({
   userId,
   updatedByUserId,
   certificationCenterId,
-  createdAt = new Date('2020-01-01'),
-  updatedAt = new Date('2023-09-12'),
+  createdAt = new Date(),
+  updatedAt = new Date(),
   disabledAt,
   isReferer = false,
   role = 'MEMBER',
 } = {}) {
   userId = _.isUndefined(userId) ? buildUser().id : userId;
   certificationCenterId = _.isUndefined(certificationCenterId) ? buildCertificationCenter().id : certificationCenterId;
+  if (disabledAt !== undefined) {
+    updatedAt = disabledAt;
+  }
```

Comment:
C'est plus idiomatique Javascript comme ça : 
```suggestion
  if (disabledAt) {
    updatedAt = disabledAt;
  }
```